src/app/header/menu/menu.component.scss

```diff
@@ -162,16 +167,56 @@ ul {
             &[href]:hover {
               color: $color-text;
             }
+
+            a {
+              padding: 0;
+            }
+
+            button {
```

Comment:
button would benefit from having `cursor: pointer` styled. And maybe some border-radius?